CommonTools/Utils/BuildFile.xml

```diff
@@ -2,7 +2,7 @@
 <use name="DataFormats/Math"/>
 <use name="FWCore/ParameterSet"/>
 <use name="FWCore/Reflection"/>
-<use name="FWCore/SOA"/>
+<use name="FWCore/SOA" source_only="1"/>
```

Comment:
@smuzaffar I get the following complaint from scram in CMSSW_11_2_X_2020-09-18-1100
```
****WARNING: Invalid tool FWCore/SOA. Please fix src/CommonTools/Utils/BuildFile.xml file.
```
Are we missing something more?

Reply:
Do you get this in latest IB?

Reply:
No, in CMSSW_11_2_X_2020-09-22-1100 there is no such complaint. Thanks!